Modules/Filtering/ImageGradient/include/itkVectorGradientMagnitudeImageFilter.hxx

```diff
@@ -155,7 +150,7 @@ VectorGradientMagnitudeImageFilter<TInputImage, TRealType, TOutputImage>::Before
   Superclass::BeforeThreadedGenerateData();
 
   // Calculate the square-roots of the component weights.
-  for (unsigned i = 0; i < VectorDimension; ++i)
+  for (unsigned i = 0; i < ImageDimension; ++i)
```

Comment:
Again, ImageDimension is not the same than VectorDimension (i.e length/size of the pixel).

Reply:
Pablo, thanks for having had a look. I'm marking as resolved following [this comment of yours](https://github.com/InsightSoftwareConsortium/ITK/pull/2279#discussion_r568480074).